<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageInfosTicket</class>
 <widget class="QWidget" name="MessageInfosTicket">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="labelUtilisateur">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Nom utilisateur</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelContenuMessage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>331</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Contenu message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateMessage">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>171</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>35</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
